e2e/_suites/fleet/fleet.go

```diff
@@ -1220,3 +1185,45 @@ func deployAgentToFleet(agentInstaller installer.ElasticAgentInstaller, containe
 
 	return cfg, agentInstaller.PostInstallFn()
 }
+
+func inputs(integration string) []kibana.Input {
+	switch integration {
+	case "linux":
+		return []kibana.Input{
+			{
+				Type:    "linux/metrics",
+				Enabled: true,
+				Streams: []interface{}{
+					map[string]interface{}{
+						"id":      "linux/metrics-linux.memory-" + uuid.New().String(),
+						"enabled": true,
+						"data_stream": map[string]interface{}{
+							"dataset": "linux.memory",
+							"type":    "metrics",
+						},
+					},
+				},
+				Vars: map[string]kibana.Var{
+					"period": {
+						Value: "1s",
+						Type:  "string",
+					},
+				},
+			}}
+	case "apm":
```

Comment:
Super super super nit: can we order the items alphabetically in the map? That would help adding/locating integrations